connection.go

```diff
@@ -338,14 +366,15 @@ func (sc *snowflakeConn) QueryContext(ctx context.Context, query string, args []
 
 	rows := new(snowflakeRows)
 	rows.sc = sc
-	rows.ChunkDownloader = populateChunkDownloader(ctx, sc, data.Data)
 	rows.queryID = sc.QueryID
 
 	if sc.isMultiStmt(&data.Data) {
 		err := sc.handleMultiQuery(ctx, data.Data, rows)
```

Comment:
`handleMultiQuery` sets up the chain of chunk downloaders
the behavior was a bit different from before, where we always created a `ChunkDownloader` and then called `handleMultiQuery` which overwrote the previous chunk downloader

Can we add method comment for `handleMultiQuery` to explain that it sets up the chain of chunk downloaders for multiple query results

Reply:
comments added